windshaft/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:vivid
```

Comment:
We have been trying to standardize on `debian:jessie` as the base image. This gives us many of the conveniences of Ubuntu, but with a overall lighter weight image.

That said, in this specific case, the majority of the container's purpose is to run Node, so we can try to reuse the `node:0.10-slim` container image as a base (this uses `debian:jessie` under the covers).

See also: https://github.com/azavea/docker-static-core/blob/7cba6b0eb3379068c33ff632508214c951d59266/Dockerfile